ingestion/setup.py

```diff
@@ -311,6 +311,7 @@
     VERSIONS["snowflake"],
     VERSIONS["elasticsearch8"],
     VERSIONS["giturlparse"],
+    "testcontainers==3.7.1",
```

Comment:
I think this is a great idea. Did not know we had that for Python too. Would be worth a small discussion with the team to see where else we can use it 🚀

Reply:
absolutely. moving forward we can also use it to setup the OM instance in the tests instead of relying on whatever's running on our local machine.

Reply:
Can you set it up globally, (i.e. you only create it once at the beginning of the test session?

Reply:
I see, It might add a bit of overhead if we need to start the server for every tests. 🤔

Reply:
in backend, we start the containers for each Test Class